### ISSUE TITLE
Alluxio files can be deleted during persisting, but the persist job was not interrupted

### ISSUE THREAD

**Issue by Yang9613** [Alluxio/alluxio#15987]
**Alluxio Version:**
2.8.0

**Describe the bug**
Alluxio files can be deleted during persisting, but the persist job was not interrupted, so the deleted files will be synced back to allxuio from UFS when the persist job is about to be completed and the delete file action happens at the same time because the file maybe already persisted to UFS  when the delete operation completed

**To Reproduce**
set writetype = ASYNC_THROUGH
1、Reproduce delete file during persisting
1) upload a large file from local by using alluxio fs copyFromLocal command in order to the persist job will not complete quickly
2) delete the uploaded file before the persist job has not been completed
2、Reproduce sync deleted file from UFS to alluxio
use a spark application overwrite the same dir repeatedly

**Comment by jja725** [comment]
yeah, I think that's a current problem with how job service handles jobs. And we are planning on a new version job service which would include a more stable Persist job.
